File: 백준/Bronze/1264. 모음의 개수/모음의 개수.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');

const 모음 = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'];

const results = [];
for(let i = 0; i < input.length; i++) {
    const str = input[i];
    
    if(str === '#') break;

    let count = 0;
    for(let j = 0; j < str.length; j++) {
        const char = str[j];
        
        if(모음.includes(char)) {
            count++;
        }
    }
    
    results.push(count);
}

console.log(results.join('\n'))

File: 백준/Bronze/10987. 모음의 개수/모음의 개수.py
str = input()

count = 0;
for i in range(len(str)):
    cur = str[i]
    
    
    if cur == 'a' or cur == 'e' or cur == 'i' or cur == 'o' or cur == 'u':
        count = count + 1;
        
print(count)